server/apps/resource/serializers.py

```diff
@@ -16,8 +18,19 @@ def validate_group(self, group):
             raise serializers.ValidationError(gettext('Group does not exist.'))
         return group
 
+    def validate(self, attrs) -> dict:
+        print("user", self.context['request'].user)
+        if Resource.objects.filter(created_by=self.context['request'].user).count() >= RESOURCE_NUMBER:
+            raise serializers.ValidationError(f"Can only create {RESOURCE_NUMBER} Resource")
```

Comment:
add `gettext` translation. 
```suggestion
            raise serializers.ValidationError(gettext(f"You can only create {RESOURCE_NUMBER} resources."))
```